src/sql/gazetteer_geometry_views.sql

```diff
@@ -25,16 +25,23 @@ from
 
 CREATE OR REPLACE RULE feature_ref_point_ins AS ON INSERT TO feature_ref_point
 DO INSTEAD
-   (
-   INSERT INTO feature (feat_type, status, description, ref_point )
-   VALUES (NEW.feat_type, 'CURR', '', NEW.ref_point );
-   INSERT INTO name (feat_id, name, status )
-   VALUES (lastval(), NEW.name, 'UNEW' )
-   RETURNING
-     feat_id,
-     gaz_preferredName(feat_id) as name,
-     (SELECT f.feat_type FROM feature f WHERE f.feat_id = feat_id),
-     (SELECT f.ref_point FROM feature f WHERE f.feat_id  = feat_id));
+    (
+    INSERT INTO feature (feat_type, status, description, ref_point )
+    VALUES (NEW.feat_type, 'CURR', '', NEW.ref_point );
+    INSERT INTO name (feat_id, name, status )
+    VALUES (lastval(), NEW.name, 'UNEW' )
+    RETURNING
+        feat_id,
+        gaz_preferredname(feat_id),
+       	(SELECT feat_type
+		FROM gazetteer.feature
+		ORDER BY feat_id DESC
+		LIMIT 1),
+  		(SELECT shape
+		FROM gazetteer.feature_geometry
```

Comment:
how does a value from the insert into `feature_ref_point` view end up being in the `feature_geometry` table ?

Reply:
Good catch. While the other views are updating feature_geometry the feature_ref_point view has a geom as part of the feature table. 

I have updated this to select from feature (as opposed to feature_geometry